src/library/scala/collection/StringOps.scala

```diff
@@ -624,10 +624,20 @@ final class StringOps(private val s: String) extends AnyVal {
     }
   }
 
+  /** Return all lines in this string in an iterator, excluding trailing line
+    *  end characters; i.e., apply `.stripLineEnd` to all lines
+    *  returned by `linesWithSeparators`.
+    *
+    * This is an alias for `lines` with a name that won't conflict with `String.lines` added in JDK 11.
```

Comment:
I don't think the wording here quite makes sense. `lines` is now deprecated, and it seems weird to me to describe the recommended method as an alias for a deprecated method.

I'd suggest simply omitting this sentence.

Reply:
Good point, I've removed this.